app/controllers/users_controller.rb

```diff
@@ -0,0 +1,45 @@
+class UsersController < ApplicationController
+  def edit
+    @user = current_user
+  end
+
+  def update_email
+    @user = User.find(current_user.id)
+
+    if @user.update(email_params)
+      redirect_to root_path, notice: 'Email изменен'
+    else
+      flash.now[:alert] = 'Не удалось изменить Email'
+      render :edit
+    end
+  end
+
+  def update_password
+    @user = User.find(current_user.id)
```

Comment:
why not @user = current_user ?
and if you do it multiple times, better move to before_action